SPECS-EXTENDED/python-iniconfig/python-iniconfig.spec

```
@@ -0,0 +1,133 @@
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+
+%undefine mariner_module_ldflags
+
+Name:               python-iniconfig
+Version:            1.1.1
+Release:            15%{?dist}
+Summary:            Brain-dead simple parsing of ini files
+# SDPX
+License:            MIT
+URL:                http://github.com/RonnyPfannschmidt/iniconfig
+BuildArch:          noarch
+BuildRequires:      python3-devel
+BuildRequires:      pyproject-rpm-macros
+BuildRequires:      python3-pip
+BuildRequires:      python3-setuptools
+BuildRequires:      python3-setuptools_scm >= 8.0.3-1
+BuildRequires:      python3-six
+BuildRequires:      python3-wheel
+
+# pytest 6+ needs this and this uses pytest for tests
+%bcond_without tests
+
+%if %{with tests}
+# We BR pytest manually to avoid a dependency on tox in ELN/RHEL
+BuildRequires:      python3-pytest
+%endif
+
+Source0:            %{pypi_source iniconfig}
+
+%global _description %{expand:
+iniconfig is a small and simple INI-file parser module
+having a unique set of features:
+
+* tested against Python2.4 across to Python3.2, Jython, PyPy
+* maintains order of sections and entries
+* supports multi-line values with or without line-continuations
+* supports "#" comments everywhere
+* raises errors with proper line-numbers
+* no bells and whistles like automatic substitutions
+* iniconfig raises an Error if two sections have the same name.}
+%description %_description
+
+
+%package -n python3-iniconfig
+Summary:            %{summary}
+%description -n python3-iniconfig %_description
+
+
+%prep
+%autosetup -n iniconfig-%{version}
+# Remove undeclared dependency on python-py
+# Merged upstream https://github.com/pytest-dev/iniconfig/pull/47
+sed -i "s/py\.test/pytest/" testing/test_iniconfig.py
+
+
+%generate_buildrequires
+%pyproject_buildrequires
+
+
+%build
+%pyproject_wheel
+
+
+%install
+%pyproject_install
+%pyproject_save_files iniconfig
+
+
+%if %{with tests}
+%check
+%pytest -v
+%endif
```

Redundant - we control running the `%check` section outside the spec.
```suggestion
%check
%pytest -v
```